### DISCUSSION TITLE
CRUD with One db session? SQLAlchemy 2.0.16

### DISCUSSION THREAD

**Discussion by Real-Music** [sqlalchemy/sqlalchemy#10467] [Usage Questions]
I want to know if it's possible to use one open db session to create a record, read the record and update or delete that record using just one db session that is open? I am using SQLAlchemy 2.0.16

**Comment by zzzeek** [comment]
hi -

this is the normal way to use the Session.    Go through the [tutorial](https://docs.sqlalchemy.org/en/20/tutorial/orm_data_manipulation.html) which illustrates a full series of operations proceeding with one Session.
